data/events/2018-detroit.yml

```diff
@@ -107,3 +107,194 @@ sponsor_levels:
     max: 4
   - id: bronze
     label: Bronze
+
+program:
+  - title: "Registration, Breakfast, Sponsors"
+    type: custom
+    date: 2018-10-03
+    start_time: "08:30"
+    end_time: "09:30"
+  - title: "Opening Welcome"
+    type: custom
+    date: 2018-10-03
+    start_time: "09:30"
+    end_time: "09:45"
+  - title: "armon-dadgar"
```

Comment:
So the Title here isn't the title for the talk but references the individual file with the title above (I'm guessing)?

Reply:
Yes, this is how it is wired together with devopsdays' hugo customizations.